opentech/apply/funds/views.py

```diff
@@ -427,6 +427,16 @@ def form_valid(self, form):
             added=added,
             removed=removed,
         )
+
+        messenger(
+            MESSAGES.PARTNERS_UPDATED_PARTNER,
+            request=self.request,
+            user=self.request.user,
+            submission=self.kwargs['submission'],
+            added=added,
```

Comment:
We can create a check above (if condition) if new partners are added then only call the messenger. SImilar for PARTNERS_UPDATED.